rascal-vscode-extension/src/lsp/RascalLSPConnection.ts

```diff
@@ -72,6 +72,9 @@ export async function activateLanguageClient(
 
 async function showContentPanel(url: string): Promise<void> {
     // dispose of old panel in case it existed
+    vscode.env.asExternalUri(vscode.Uri.parse(url))).then(freshURL => {
+
```

Comment:
why is this one empty?

Reply:
Something very wrong happened with a `git stash` or with a merge. I'll have to code this again.